easy-ocw/ocw-pip-dependencies.txt

```diff
@@ -9,3 +9,4 @@ esgf-pyclient>=0.1.6
 python-dateutil>=2.5.3
 mock>=2.0.0
 myproxyclient>=1.4.3
+podaacpy>=1.0.1
```

Comment:
Once this is upgraded to 1.0.2 we are good to merge this patch. Thanks @Omkar20895